apps/src/templates/projects/NewProjectButtons.jsx

```diff
@@ -52,12 +53,12 @@ const PROJECT_INFO = {
     label: i18n.projectTypeFrozen(),
     thumbnail: "/shared/images/fill-70x70/courses/logo_frozen.png"
   },
-  'mc': {
-    label: i18n.projectTypeMC(),
+  'minecraft_adventurer': {
```

Comment:
if you want to change these ids, you'll also have to update the project widget section of the level-builder script edit page, which allows level-builders to select a specific sub-set.

Reply:
Also, this controls the project link - did they change to /p/minecraft_adventurer?

Reply:
yep, they did change to /projects/minecraft_adventurer, which is why I changed these. I'll take a look at levelbuilder.

Reply:
I've made the change in levelbuilder, but i'm flying blind because the projects widget is not showing up for me in stage extras on localhost. This is not a regression, since I see the same thing when I check out the staging branch locally.